### ISSUE TITLE
Deleting project on edge throws exception

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#1935]
```
I, [2019-02-13T16:23:20.110153 #12006]  INFO -- : [5b5bf61a-dc1c-4064-9ada-60e86ba412ae] Started DELETE "/api/v1/projects/33d8cb5c-d6e9-4e63-9d06-0607a585e77d" for 127.0.0.1 at 2019-02-13 16:23:20 -0800
I, [2019-02-13T16:23:20.112499 #12006]  INFO -- : [5b5bf61a-dc1c-4064-9ada-60e86ba412ae] Processing by Api::V1::ProjectsController#destroy as */*
I, [2019-02-13T16:23:20.112729 #12006]  INFO -- : [5b5bf61a-dc1c-4064-9ada-60e86ba412ae]   Parameters: {"id"=>"33d8cb5c-d6e9-4e63-9d06-0607a585e77d"}
a65ebe18-e06c-4405-8cb0-d142eea8fb18) to Sidekiq(default)
I, [2019-02-13T16:23:20.291675 #12006]  INFO -- : [5b5bf61a-dc1c-4064-9ada-60e86ba412ae] Completed 500 Internal Server Error in 179ms (ActiveRecord: 54.2ms)
F, [2019-02-13T16:23:20.295542 #12006] FATAL -- : [5b5bf61a-dc1c-4064-9ada-60e86ba412ae]
F, [2019-02-13T16:23:20.295866 #12006] FATAL -- : [5b5bf61a-dc1c-4064-9ada-60e86ba412ae] ActiveRecord::ActiveRecordError (cannot touch on a new or destroyed record object. Consider using persisted?, new_record?, or destroyed? before touching):
F, [2019-02-13T16:23:20.295981 #12006] FATAL -- : [5b5bf61a-dc1c-4064-9ada-60e86ba412ae]
F, [2019-02-13T16:23:20.296077 #12006] FATAL -- : [5b5bf61a-dc1c-4064-9ada-60e86ba412ae] app/models/event.rb:113:in `touch_project!'
[5b5bf61a-dc1c-4064-9ada-60e86ba412ae] app/controllers/api/v1/projects_controller.rb:50:in `destroy'
[5b5bf61a-dc1c-4064-9ada-60e86ba412ae] lib/middleware/omniauth_stack.rb:10:in `call'
```

**Comment by SMaxOwok** [comment]
Yeah, really need to sort this thing out.

**Comment by SMaxOwok** [comment]
Closed with #1803
